src/util.cpp

```diff
@@ -243,8 +244,13 @@ int LogPrintStr(const std::string &str)
         }
 
         // Debug print useful for profiling
-        if (fLogTimestamps && fStartedNewLine)
-            ret += fprintf(fileout, "%s ", DateTimeStrFormat("%Y-%m-%d %H:%M:%S", GetTime()).c_str());
+        if (fLogTimestamps && fStartedNewLine) {
+            std::locale loc(std::locale::classic(), new boost::posix_time::time_facet("%Y-%m-%dT%H:%M:%sZ"));
+            std::stringstream ss;
+            ss.imbue(loc);
+            ss << boost::posix_time::microsec_clock::universal_time();
+            ret += fprintf(fileout, "[%s] ", ss.str().c_str());
+        }
```

Comment:
What is the reason for this? Is this required to achieve your PRs goal or is this another feature that is unrelated to this PR?